scripts/run-integration-tests.sh

```diff
@@ -173,98 +132,80 @@ export KUBECONFIG=$KUBECONFIG_PATH
 # Use should_execute in common.sh to short circuit methods if $TEST_PASS -eq 1
 
 add_helm_repo
-install_helm_chart base
-ensure_controller_pods
+BASE_INTEGRATION_DURATION=0
 if [[ "$TEST_PASS" -ne 0 ]]; then
   echo "NOTE: Skipping base test run because test is marked as failed"
+elif [[ -z "${BASE_GIT_TAG+x}" ]]; then
+  echo "NOTE: Skipping base test run because BASE_GIT_TAG is not given"
 else
-  echo "Running base integration test"
-  # TODO: RUN BASE TEST HERE
+  echo "*******************************************************************************"
+  echo "Running integration test setup on BASE_GIT_TAG, $BASE_GIT_TAG"
+  echo ""
+
+  __ack_source_tmpdir="/tmp/ack-src-$BASE_GIT_TAG"
+  echo "Checking out ack source code for $BASE_GIT_TAG ..."
+  git clone git@github.com:varun1524/aws-service-operator-k8s.git "$__ack_source_tmpdir" || exit 1
+
+  pushd "$__ack_source_tmpdir" || exit
+  git checkout -b "$BASE_GIT_TAG" "$BASE_GIT_TAG"
+
+  __test_crd_path="/tmp/crd/base"
+  ensure_service_controller_running "$BASE_GIT_TAG" "$__test_crd_path"
```

Comment:
here, you should pass the name of the service as a parameter to `ensure_service_controller_running`